### PR TITLE
Home->Team BE integration

### PR DESCRIPTION
**Pull request by ice-orion**

### CODE REVIEW

File: src/screens/HomeFlow/Home/components/Team/components/TeamMember.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import {Touchable} from '@components/Touchable';
+import {COLORS} from '@constants/colors';
+import {LogoIcon} from '@svg/LogoIcon';
+import {font} from '@utils/styles';
+import React, {memo} from 'react';
+import {StyleSheet, Text, View} from 'react-native';
+import {rem} from 'rn-units';
+
+type Props = {
+  username?: string | null;
```
Comment (ice-ares) [inline src/screens/HomeFlow/Home/components/Team/components/TeamMember.tsx:13]: never null

File: src/screens/HomeFlow/Home/components/Team/components/TeamMember.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  profilePictureUrl?: string;
```
Comment (ice-ares) [inline src/screens/HomeFlow/Home/components/Team/components/TeamMember.tsx:14]: never null

File: src/screens/HomeFlow/Home/components/Team/components/TeamMember.tsx (hunk 1 above)
Comment (ice-orion) [reply src/screens/HomeFlow/Home/components/Team/components/TeamMember.tsx]: changed User type -> `id`, `username` and `profilePictureUrl` are now always present.

File: src/screens/HomeFlow/Home/components/Team/components/TeamMember.tsx (hunk 2 above)
Comment (ice-orion) [reply src/screens/HomeFlow/Home/components/Team/components/TeamMember.tsx]: changed User type -> `id`, `username` and `profilePictureUrl` are now always present.

**Review by ice-ares — APPROVED** [review]

**Review by ice-hector — APPROVED** [review]

**Review by ice-eros — APPROVED** [review]

**Pull request merged**